src/support/steps/general.ts

```diff
@@ -247,7 +247,7 @@ When('I clear cache', async function (this:ICustomWorld) {
 
     this.sections.set('dashboard');
     await this.sections.toggle('clearCacheBtn');
-    await this.page.waitForLoadState('load', { timeout: 30000 });
+    await expect(this.page.getByText('WP Rocket: Cache cleared.')).toBeVisible();
```

Comment:
It might be good to add a timeout here:

toBeVisible({ timeout: 30000 });

In case there is an issue, at least the test will fail after 30s, otherwise I think it could hang forever 🤔

Reply:
The default timeout for expect is [5 seconds](https://playwright.dev/docs/test-timeouts#:~:text=Timeouts-,Timeouts,-Introduction%E2%80%8B) then it will fail. but yes, we can change to 30s for bad internet connection.

Reply:
Oh right! If 5s works for you, maybe it's enough.

Reply:
It's working so far :smile:  Thank you :pray:

Reply:
I think in general for safety, if 5 s is small, we can later set expected timeout in config as mentioned here https://playwright.dev/docs/test-timeouts#set-expect-timeout-in-the-config to avoid manually adding it in n steps. What do you think? Shall we do this here?

Reply:
I'd say let's keep it like this for now and if some timeout occurs, we will adjust the config file

Reply:
After further investigation, changing the asserted msg to see how long will it take to timeout, and seems it's 30s by default there
![Screenshot from 2024-08-19 21-18-24](https://github.com/user-attachments/assets/334f3f99-9e64-46dc-899f-470e7bc4887c)
And as per discussion with Mike, seems we aren't using a test runner of a playwright but a cucumber test runner... (can confirm that if we used the playwright-suggested timeout set to 40s in hooks.ts, it won't work)

Reply:
As per discussion with Mike, it's hard now to use the playwright method for global expect timeout as we are using a cucumber test runner. In fact, can confirm that using code mentioned at the playwright material won't work and it will fail after 5s by default unless we hardcoded the 30s inside tobevisible.

Recommend investigate this later in other GH , may be https://github.com/wp-media/wp-rocket-e2e/issues/39